web/src/main/client/src/components/toggle/Toggle.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Toggle extends Component {
+
+  constructor(props){
+    super(props);
+
+    this.state = {
+      hidden: props.hidden
+    }
+
+    this.handleHide = this.handleHide.bind(this);
+  }
+
+  handleHide() {
+    const url = `/admin/posts/${this.props.uuid}/hide`;
```

Comment:
Toggling a post does not work. I guess it is because this line makes the toggle request go the current address instead of the API.